sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -160,8 +161,8 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     // An Access policy.
     struct AccessPolicy
     {
-      std::string Start; // The date-time the policy is active.
-      std::string Expiry; // The date-time the policy expires.
+      Core::DateTime Start; // The date-time the policy is active.
+      Core::DateTime Expiry; // The date-time the policy expires.
```

Comment:
We ever mentioned during review meeting that these should be something like XxxOn? Can you check .Net SDK?

Reply:
But that should be another separate PR.

Reply:
This comment applies to all other member names

Reply:
Modified all occurrencies.